packages/cursorless-engine/src/core/StoredTargets.ts

```diff
@@ -1,10 +1,25 @@
+import { DefaultMap } from "@cursorless/common";
 import { Target } from "../typings/target.types";
 
 /**
  * Used to store targets between commands.  This is used by marks like `that`
  * and `source`.
  */
-export class StoredTargets {
+export class StoredTargetMap {
+  private targetMap: DefaultMap<string, StoredTargets> = new DefaultMap(
```

Comment:
string? STRING?? I assume that is for future bookmarks, but is there anyway we could type our known stored targets?

Reply:
Haha yeah that's for custom bookmarks. I could just add separate fields for `that` and `source` and then the string getter would be just for custom use?

Reply:
Sounds good!

Reply:
Ok for now I just made it strongly typed.  We can handle custom bookmarks when we get there